keras_cv/bounding_box/converters.py

```diff
@@ -25,6 +25,25 @@ class RequiresImagesException(Exception):
     pass
 
 
+def clip_boxes(boxes, box_format, image_shape):
```

Comment:
keras_cv.bounding_box.clip_to_image should do this, right?
(from bounding_box/utils.py)

Reply:
(If not, and this needs to be a separate method, maybe it should also live in `utils.py` instead of `converters.py`?)

Reply:
Ideally it does, but that would probably error out given:
"`bounding_boxes` must include a class_id index on the final "
-- as discussed last time, this is a huge tech debt we should get rid of, but I don't want it to block the current development as well.

Reply:
Gotcha. If that method doesn't work as intended and it's not simple to update it, I think we should keep this version of it internal to the training script.

Otherwise we have two separate but nearly identical utilities.

Reply:
Done.